src/robot_dart/gui/magnum/gs/camera.cpp

```diff
@@ -69,6 +69,14 @@ namespace robot_dart {
 #endif
                 }
 
+                Magnum::Matrix3 intrinsicMatrix()
+                {
+                    Magnum::Matrix4 perspProj = Magnum::Matrix4::perspectiveProjection(_fov, _aspect_ratio, _near_plane, _far_plane);
```

Comment:
Why not using the `_camera->projectionMatrix()`? See [here](https://doc.magnum.graphics/magnum/classMagnum_1_1SceneGraph_1_1Camera.html#a21f5b5c5f4aa7726a6343328e065aa43).